util/text/text.go

```diff
@@ -116,63 +117,13 @@ func wordLen(word string) int {
 		if char == '\x1b' {
 			escape = true
 		}
-
 		if !escape {
 			length += runewidth.RuneWidth(rune(char))
 		}
-
 		if char == 'm' {
 			escape = false
 		}
 	}
 
 	return length
 }
-
-// splitWord split a word at the given length, while ignoring the terminal escape sequences
```

Comment:
Even if it's not used anymore, let's keep that function (and the corresponding tests) around for now. It's working and might be useful in the future.